app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -278,6 +279,22 @@ def volumes
     end
   end
 
+  # Store VPC volume profiles in cloud_volume_types table. Use name for the ems_ref and name. Use the family as description.
+  # @return [void]
+  def cloud_volume_types
+    $ibm_cloud_log.info('Starting cloud volume types persistance.')
+    collector.volume_profiles.each do |v|
+      $ibm_cloud_log.info("Saving #{v} to persister.")
```

Comment:
At most this should be to `.debug`, not sure we need the content of each individual volume_profile in the logs even to debug though

Reply:
Yeah, that wasn't supposed to be pushed. It was for testing.